app/routines/freeFlight/views/Fly.js

```diff
@@ -104,8 +104,8 @@ define(['app', 'backbone', 'JST', 'q', 'leaflet-dist', 'bootstrap-slider', 'unde
                 // Detect when system has landed, then instruct disarm.
                 this.model.platform.on('status:standby', function() {
```

Comment:
So, there isn't any event that triggers the `status:standby` event on the platform.  I think we should make that event work before merging this, otherwise we end up with no "done" button.